src/RDSParser.cpp

```diff
@@ -18,12 +18,14 @@
 #define DEBUG_FUNC0(fn)          { Serial.print(fn); Serial.println("()"); }
 
 /// Setup the RDS object and initialize private variables to 0.
-RDSParser::RDSParser() {
+RDSParser::RDSParser()
+{

```

Comment:
Here and in other places re re-formatting should be reversed to focus on the changes.